ipfailover/keepalived/lib/failover-functions.sh

```diff
@@ -34,6 +34,23 @@ function cleanup() {
   exit 0
 }
 
+function unconfigure_failover() {
+  echo "  - Removing ip_vs module ..."
+  modprobe -r ip_vs
+
+  echo "  - Removing keepalived multicast iptables rule ..."
+  chain="${HA_IPTABLES_CHAIN:-""}"
+  if [[ -n ${chain} ]]; then
+    if iptables -S | grep 224.0.0.18 > /dev/null 2>&1 ; then
+      echo "  - Removing keepalived multicast iptables rule ..."
+      iptables -D ${chain} 1 -d 224.0.0.18/32 -j ACCEPT
+    fi
+  fi
+
+  cleanup $(pidof /usr/sbin/keepalived)
+  service keepalived stop
```

Comment:
That takes me back!  Any reason not to use systemctl?  Last time I used the service command (which was years ago, on RHEL7), it worked, but it printed some extra noise encouraging using systemctl instead.

Reply:
since cleanup function is will kill keepalived and exit there is no need to systemctl or remove the config file so I will remove those two lines